GomokuClient/packages/gomoku-core/src/pages/JoinGame.tsx

```diff
@@ -122,9 +108,9 @@ const JoinGame = ({ gameHistory, playerID }: JoinGameProps) => {
                   <GameTimeMobile
                     opponentView
                     {...commonGameTimeProps}
-                    player={players[0]}
+                    player={players.black}
```

Comment:
Ideally the player with whom you play should be on top and youself on bottom. Not sure, if we should cover this in that patch, but makes sense to attach this players values based on that. In theory we can calculate your position with current socket events data

Reply:
Yes, I agree, that would be perfect, but let's cover this in different patch